server/graphql_schemas/attend/schema.py

```diff
@@ -0,0 +1,65 @@
+import graphene
+from api.models import Attend, Event, GoogleUser
```

Comment:
Imports should follow PEP should follow PEP8 style guide https://www.python.org/dev/peps/pep-0008/#imports